__tests__/index.test.js

```diff
@@ -100,4 +101,68 @@ describe("for CimpressTranslationsClient", () => {
       }
     });
   });
+
+  describe("for patchStructure()", () => {
+    afterEach(nock.cleanAll);
+
+    it("returns the successful response", async () => {
+      let n = nock(TEST_URL)
+        .patch(route => route.match(pope(API.v1ServicesIdStructure, {id: TEST_ID})), TEST_STRUCTURE)
```

Comment:
This part tests the simplified PATCH call for adding a field but not the full syntax, e.g. `{ "op": "add", "variable": "count", "value": 1 }`. I would guess that it might be used for deleting keys as well.